index.js

```diff
@@ -99,6 +105,10 @@ class Ky {
 			...otherOptions
 		};
 
+		if (supportsAbortController && !this._options.signal) {
```

Comment:
We still want timeout to work, even if the user sets their own signal.

Reply:
ookay....